eat.go

```diff
@@ -11,10 +11,11 @@ type Eat struct {
 	UEID          *UEID          `cbor:"11,keyasint,omitempty" json:"ueid,omitempty"`
 	Origination   *StringOrURI   `cbor:"12,keyasint,omitempty" json:"origination,omitempty"`
 	OemID         *[]byte        `cbor:"13,keyasint,omitempty" json:"oemid,omitempty"`
-	SecurityLevel *SecurityLevel `cbor:"14,keyasint,omitempty" json:"security-level,omitempty"`
-	SecureBoot    *bool          `cbor:"15,keyasint,omitempty" json:"secure-boot,omitempty"`
+	SecurityLevel *SecurityLevel `cbor:"14,keyasint,omitempty" json:"seclevel,omitempty"`
+	SecureBoot    *bool          `cbor:"15,keyasint,omitempty" json:"secboot,omitempty"`
```

Comment:
we should keep the scope of this PR to the profile claim only -- these two changes are likely to have a fallback on other unit tests...

Reply:
ok, will remove.